<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" colspan="3">
     <widget class="QFrame" name="f_png">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lbl_png">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lbl_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="f_login">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lbl_sesion">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Inicio de Sesion</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_usuario">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Usuario:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_username">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_contra">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Contraseña:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_contra">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn_ingresar">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NAVEGAR</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QFrame" name="f_relleno1">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="lbl_fecha">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>FECHA</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="f_acciones">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_sesion_2">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>¿Que desea realizar?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rb_inventario">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ver inventario</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="rb_entrada">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nueva entrada</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="radioButton_3">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Salida</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QFrame" name="f_relleno2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tab_copeco">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_inventario">
          <attribute name="title">
           <string>INVENTARIO</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_salidas">
          <attribute name="title">
           <string>SALIDAS</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_entradas">
          <attribute name="title">
           <string>ENTRADAS</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
